hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeysDeleteResponse.java

```diff
@@ -29,32 +29,33 @@
 
 import jakarta.annotation.Nonnull;
 import java.io.IOException;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.BUCKET_TABLE;
 import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.DELETED_TABLE;
 import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.KEY_TABLE;
-import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.OPEN_KEY_TABLE;
 import static org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.Status.OK;
 import static org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.Status.PARTIAL_DELETE;
 
 /**
  * Response for DeleteKey request.
  */
-@CleanupTableInfo(cleanupTables = {KEY_TABLE, OPEN_KEY_TABLE, DELETED_TABLE, BUCKET_TABLE})
+@CleanupTableInfo(cleanupTables = {KEY_TABLE, DELETED_TABLE, BUCKET_TABLE})
```

Comment:
Why OPEN_KEY_TABLE is removed here？ Could both OPEN_KEY_TABLE and OPEN_FILE_TABLE included here？

Reply:
Now we are not deleting from `OPEN_KEY_TABLE` here which was added in https://github.com/apache/ozone/pull/6079. Instead using `OpenKeyCleanupService` to do the same. Still it is required?

Reply:
This @CleanupTableInfo is to cleanup the table cache after the corresponding transactions flushed to RocksDB. In the delete, we put the empty entry in openKeyTable cache to represent key is deleted from the RocksDB. 
```
   // Remove the open key by putting a tombstone entry
      openKeyTable.addCacheEntry(dbOpenKey, trxnLogIndex);
```
But now we alter the key metadata instead of delete the key. So shall we still put an empty entry here?  This also reminds me we may need check the output of CLI command which list the openKeys, we need to tell if the key is marked for deleted or not.

Reply:
Removed adding empty cache entry here.
Okay I will raise Jira to update CLI for displaying whether `openKey` is marked for deletion or not.

Reply:
@ashishkumar50 , I don't mean remove the cache entry adding, instead I think we should add the cache entry with update the keyInfo, instead of an empty entry currently.

Reply:
Added with updated `keyInfo.`